test/constructor.spec.js

```diff
@@ -56,6 +56,16 @@ describe('ipfs-api constructor tests', () => {
       clientWorks(ipfsAPI(splitted[2], splitted[4]), done)
     })
 
+    it('specify host, port and api path', (done) => {
+      const splitted = apiAddr.split('/')
+
+      clientWorks(ipfsAPI({
+        host: splitted[2],
+        port: splitted[4],
+        'api-path': '/api/v0/'
```

Comment:
Since this is the default, this test doesn't really prove anything except for you can pass `api-path` as an option without it breaking.

Ideally we'd actually setup a proxy for the test and point the API at the proxy with `api-path: '/ipfs/api/v0'`. Obviously this wouldn't be possible in the browser so you'd need to skip.

I'd accept a PR for that!